ui/src/app/views/error-handler/component.ts

```diff
@@ -0,0 +1,27 @@
+// UI Component framing the overall app (title and nav).
+// Content is in other Components.
+
+import {ChangeDetectorRef, Component, OnInit} from '@angular/core';
+import 'clarity-icons';
+@Component({
+  selector: 'app-error-handler',
+  styleUrls: ['./component.css'],
+  templateUrl: './component.html'
+})
+export class ErrorHandlerComponent implements OnInit {
+  notifyFiveHundred = false;
+
+  constructor() {}
+
+  ngOnInit(): void {
+    window['handleFiveHundred'] = () => {
+      this.notifyFiveHundred = true;
+      setTimeout(() => {
+        this.notifyFiveHundred = false;
+      }, 10000);
```

Comment:
This auto-dismisses the dialog after 10s, right?

I think unless we're still retrying and succeed, auto-dismissing the dialog will be confusing.

Reply:
Okay, I can make that switch. I know when I'm on a webpage, I always get annoyed if my screen starts cluttering up with a bunch of dialogs, but I don't think that will happen here.